timesketch/lib/sigma_util.py

```diff
@@ -133,6 +141,14 @@ def get_sigma_rules(rule_folder, sigma_config=None):
                     continue
 
                 rule_file_path = os.path.join(dirpath, rule_filename)
+                block_because_csv = False
+
+                if any(x in rule_file_path for x in ignore_list):
+                    block_because_csv = True
+
+                if block_because_csv:
```

Comment:
You can get rid of this block, and just `continue` in the `if any()` above. block_because_csv is not used anywhere else it seems.

Reply:
done